docs/libcurl/curl_url_set.3

```diff
@@ -75,7 +75,7 @@ If used in with \fICURLU_APPENDQUERY\fP, the provided part will be appended on
 the end of the existing query - and if the previous part didn't end with an
 ampersand (&), an ampersand will be inserted before the new appended part.
 
-When \fCURLU_APPENDQUERY\fP is used together with \fICURLU_URLENCODE\fP,
+When \fCCURLU_APPENDQUERY\fP is used together with \fICURLU_URLENCODE\fP,
```

Comment:
I think this should be `\fICURLU_APPENQUERY\fP` to match the others?

Reply:
Ah yeah, of course.